doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,27 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows both the application core and the network core to be updated at the same run.
+
+To enable the simultaneous update of multiple images set the following options:
+
+* :kconfig:`CONFIG_BOOT_UPGRADE_ONLY` - The simultaneous update of multiple images does not support network core image reversion so it is recommended to disable application image reversion.
+* :kconfig:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
+* :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple updatatable images by setting this option to ``2``.
+
+Additionally the memory partitions for secondary images for both the application and the network core must be defined.
+For the example of such configuration refer to :ref:`ug_thingy53`.
+
+FW update binaries container
+============================
+
+The build system will automatically place both the application core and network core update binaries (:file:`app_update.bin` and :file:`net_core_app_update.bin`) into a container package named :file:`dfu_application.zip`.
+This container package can be used by update tools to pass both images during the process of the simultaneous update of multiple images.
```

Comment:
```suggestion
This container package can be used by update tools to pass both images during the simultaneous update of multiple images.
```
"can be used" or "is used"? I was under the impression that the container is the only option for multi-image update.